docker/Dockerfile

```diff
@@ -77,10 +77,6 @@ RUN ln -s /var/lib/runit/service/nginx /etc/service/nginx
 ADD dist /opt/spark/dist
 ADD krb5.conf.mustache /etc/
 
-# Add external jar for pushing metrics via StatsD (https://github.com/vidhyaarvind/spark-statsd)
-# current version: Spark 2.2.1, Scala 2.11
-ADD jars/spark-statsd-2.2.1-2.11.jar /opt/spark/dist/jars/  
```

Comment:
@soumasish what was the motivation for removing this?

Reply:
Removed it accidentally, added it back.